### DISCUSSION TITLE
[Feature] Arrow key navigation for selection a suggestion when typing a name

### DISCUSSION THREAD

**Discussion by eygraber** [immich-app/immich#5398] [Feature Request]
### The feature

When typing a name for a person, the list of suggestions isn't navigable with arrow keys. For example if I have people named Foo1, Foo2, and Foo3, if I type Foo, and I want to select Foo3, I'd like to be able to use the down arrow key to navigate to it. Currently I can use tab, but that's not as natural, and also requires first navigating past the Done button.

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by jrasm91** [comment]
Implemented in #5486
